cmd/hlb/command/lint.go

```diff
@@ -8,47 +8,56 @@ import (
 	"io/ioutil"
 	"os"
 
-	"github.com/openllb/hlb/builtin"
+	"github.com/moby/buildkit/client"
+	"github.com/openllb/hlb"
 	"github.com/openllb/hlb/checker"
 	"github.com/openllb/hlb/diagnostic"
 	"github.com/openllb/hlb/errdefs"
 	"github.com/openllb/hlb/linter"
-	"github.com/openllb/hlb/parser"
-	"github.com/openllb/hlb/pkg/filebuffer"
 	cli "github.com/urfave/cli/v2"
 )
 
 var lintCommand = &cli.Command{
 	Name:      "lint",
 	Usage:     "lints a hlb module",
-	ArgsUsage: "<*.hlb>",
+	ArgsUsage: "<uri>",
 	Flags: []cli.Flag{
 		&cli.BoolFlag{
 			Name:  "fix",
 			Usage: "write module with lint errors fixed and formatted to source file",
 		},
 	},
 	Action: func(c *cli.Context) error {
-		rc, err := ModuleReadCloser(c.Args().Slice())
+		uri := DefaultHLBFilename
+		if c.NArg() > 1 {
+			return fmt.Errorf("expected at most 1 arg but got %d", c.NArg())
```

Comment:
Maybe best to show the usage here?

Reply:
Does `urfave/cli` have an API of showing usage? Or do you mean something like the error message showing: `hlb lint [ <uri> ]`?

Reply:
I'm thinking of `ShowCommandHelp`.